src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final String FORMAT_STRING = "dd.MM.yyyy";
+    private static final String DELIMITER = " ";
+    private static final int DATA_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOUR_INDEX = 2;
+    private static final int SALARY_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder();
+        DateTimeFormatter dateParserFormatter = DateTimeFormatter.ofPattern(FORMAT_STRING);
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/211301525-16f2c0f2-adfc-45ca-9155-c890a7413834.png)
It's better to make a formatter as a constant class field